praktikumsplaner-backend/src/main/java/de/muenchen/oss/praktikumsplaner/dtos/CreateMeldezeitraumDTO.java

```diff
@@ -0,0 +1,17 @@
+package de.muenchen.oss.praktikumsplaner.dtos;
+
+import de.muenchen.oss.praktikumsplaner.annotations.StartDateBeforeEndDate;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import java.time.LocalDate;
+import lombok.Builder;
+
+@Builder
+@StartDateBeforeEndDate(startDate = "startZeitpunkt", endDate = "endZeitpunkt", message = "Startdatum muss vor dem Enddatum liegen.")
+public record CreateMeldezeitraumDTO(
+        @NotNull
+        @Size(max = 255, message = "Name für Zeitraum darf nicht länger als {max} Zeichen sein.")
+        String zeitraumName,
```

Comment:
the rules doesnt match the frontend. In the frontend I can send a request when the name is blank (no string or just spaces).

Via Postman i created an Entity without a name:

POST http://localhost:39146/meldezeitraum
RequestBody:  
```
{
    "zeitraumName": "",
    "startZeitpunkt": "2023-01-03",
    "endZeitpunkt": "2023-01-02"
}
```

ResponseBody:

```
{
    "id": "e4108594-3cae-4d5f-8feb-ab576eaf20e2",
    "zeitraumName": "",
    "startZeitpunkt": "2023-01-01",
    "endZeitpunkt": "2023-01-02"
}
```